features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -21,17 +21,22 @@ async function readJSONFile(relativeFilePath, zipFile) {
     return readFullPathJSONFile(entry);
 }
 
+function decode(s) {
+    let d = new TextDecoder();
+    let a = s.split("").map((r) => r.charCodeAt());
+    return d.decode(new Uint8Array(a));
+}
+
 async function readFullPathJSONFile(entry) {
     const rawContent = await entry.getContent();
-    const fileContent = new TextDecoder("utf-8").decode(rawContent);
-
+    const fileContent = new TextDecoder().decode(rawContent);
     if (!fileContent) {
         throw new MissingContentImportException(entry._id);
     }
 
     return JSON.parse(fileContent, (key, value) => {
         if (typeof value === "string") {
-            return decodeURIComponent(escape(value));
+            return decodeURIComponent(decode(value));
```

Comment:
OK, this is just weird... Previously, we were calling `escape` on `value`,  which would actually URL encode the damn thing :D Then we would immediately URL decode it again. I bet there was some logic behind it, but I can only guess: Was it one way of getting some consistent encoding for `value`?

Anyway, now run `value` through `TextDecoder` instead. So is the `decodeURIComponent` call still necessary? Not sure how this could end up being URL encoded, since it's being read from a file. Previously, it ended up that way because of the `escape` call, I presume.

Reply:
Right, it's weird, but only because Facebook encoding is weird...

Reply:
Yeah seems like we don't need decodeURIComponent any longer